docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -95,7 +95,7 @@ Expect the following output:
 serviceaccount/yugabyte-platform-universe-management created
 ```
 
-You need to grant access to this ServiceAccount using ClusterRoles and Roles, as well as ClusterRoleBindings and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you.
+You need to grant access to this service account using ClusterRoles and Roles, as well as ClusterRoleBindings and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you.<br><br>
 Ensure that you have replaced the `namespace` from the commands with the correct namespace of the previously created ServiceAccount.
```

Comment:
a bit confusing; maybe something like "Replace `namespace` in the commands that follow with the namespace of the service account you created."
And in the commands, is `<SA_NAMESPACE>` different from `<namespace>`?

Reply:
Rephrased.
sa_namespace is the one that was created previously. So, as far as I understand, if the service account's namespace was yb_sa, the command would be:

```
| sed "s/namespace: yb_sa/namespace: yb_sa"/g \
  | kubectl apply -n yb_sa -f -
```